packages/protocol/contracts/interfaces/IMoxieBondingCurveLegacyV1.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.24;
+
+interface IMoxieBondingCurveLegacyV1 {
+    struct FeeInput {
+        uint256 protocolBuyFeePct;
+        uint256 protocolSellFeePct;
+        uint256 subjectBuyFeePct;
+        uint256 subjectSellFeePct;
+    }
+
+    function initializeSubjectBondingCurve(
```

Comment:
Do we need this interface? Its added back in referral branch

Reply:
my bad for got to remove the legacy interfaces, removing them quickly

Reply:
done @0xsarvesh